common/skip links/reports/pre-ally-1.json

```
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/GCWeb/sites/skiplinks/skiplinks-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Skip links",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "gc-features"
+        }
+    },
+
+    "earl:assertedBy": {
+        "foaf:name": "Service Canada - Principal Publisher",
+        "foaf:homepage": "https://github.com/ServiceCanada",
+        "@type": ["earl:Assertor", "foaf:Organization"],
+        "earl:mainAssertor": {
+            "foaf:name": "Hongbin Yu (Github: @hongbinyu413)",
+            "foaf:homepage": "https://github.com/hongbinyu413",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+
+    "dct:date": "2023-07-06",
+
+    "dct:description": "Evaluation of all WCAG 2.1 SC at level AA.",
+
+    "dct:source": "act:rulesets/wcag2x/wcag21_all_levelAA.json",
+    "acr:standard": "https://www.w3.org/TR/WCAG21",
+    "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+
+    "earl:result": [{
+            "earl:test": "WCAG21:non-text-content",
+            "earl:outcome": "earl:passed",
```

```suggestion
            "earl:outcome": "earl:inapplicable",
```
There is no non-text-content